specification/cognitiveservices/data-plane/CustomVision/Prediction/stable/v3.0/Prediction.json

```diff
@@ -0,0 +1,917 @@
+            "name": "imageData",
+            "in": "formData",
+            "description": "Binary image data.",
+            "required": true,
+            "type": "file"
+          },
+          {
+            "$ref": "#/parameters/ApiKey"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/ImagePrediction"
+            }
+          },
+          "default": {
+            "description": "Error response",
+            "schema": {
+              "$ref": "#/definitions/CustomVisionError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Successful DetectImageWithNoStore request": {
+            "$ref": "./examples/DetectImageWithNoStore.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "ImageUrl": {
+      "description": "Image url.",
+      "type": "object",
+      "properties": {
+        "url": {
+          "description": "Url of the image.",
+          "type": "string",
+          "x-nullable": true
+        }
+      },
+      "x-nullable": true
+    },
+    "ImagePrediction": {
+      "description": "Result of an image prediction request.",
+      "type": "object",
+      "properties": {
+        "id": {
+          "format": "uuid",
+          "description": "Prediction Id.",
+          "type": "string",
+          "readOnly": true,
+          "x-nullable": false
+        },
+        "project": {
+          "format": "uuid",
+          "description": "Project Id.",
+          "type": "string",
+          "readOnly": true,
+          "x-nullable": false
+        },
+        "iteration": {
+          "format": "uuid",
+          "description": "Iteration Id.",
+          "type": "string",
+          "readOnly": true,
+          "x-nullable": false
+        },
+        "created": {
+          "format": "date-time",
+          "description": "Date this prediction was created.",
+          "type": "string",
+          "readOnly": true,
+          "x-nullable": false
+        },
+        "predictions": {
+          "description": "List of predictions.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Prediction"
+          },
+          "readOnly": true,
+          "x-nullable": true
+        }
+      },
+      "x-nullable": true
+    },
+    "Prediction": {
+      "description": "Prediction result.",
+      "type": "object",
+      "properties": {
+        "probability": {
+          "format": "float",
+          "description": "Probability of the tag.",
+          "type": "number",
+          "readOnly": true,
+          "x-nullable": false
+        },
+        "tagId": {
+          "format": "uuid",
+          "description": "Id of the predicted tag.",
+          "type": "string",
+          "readOnly": true,
+          "x-nullable": false
+        },
+        "tagName": {
+          "description": "Name of the predicted tag.",
```

Comment:
Do we have a list of tagNames?

Reply:
The user creates the tag names using the training api in the project and this api returns them.